example/presto_standalone/main.tf

```diff
@@ -3,21 +3,21 @@ locals {
   nomad_datacenters = ["dc1"]
 
   presto = {
-    shared_secret_provider   = "user"
-    vault_kv_policy_name     = "kv-secret"
-    vault_kv_path            = "secret/data/presto"
-    vault_kv_secret_key_name = "cluster_shared_secret"
-    service_name             = "presto"
+    use_vault_secret_provider = true
+    vault_kv_policy_name      = "kv-secret"
+    vault_kv_path             = "secret/data/presto"
```

Comment:
Could you put creds, not in the root of kv storage? [example](https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/example/standalone-vault-provided-creds/main.tf#L18)

Reply:
Sure. May I ask why?

Reply:
Makes easy to setup granular access later. [Example with consul kv](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/ansible/templates/conf/consul/policies/producer_policy.hcl#L1-L9) but principle is the same

Reply:
Ok, will fix